engines/python/setup/djl_python/tensorrt_llm.py

```diff
@@ -110,9 +141,15 @@ def inference(self, inputs):
 _service = TRTLLMService()
 
 
-def handle(inputs: Input):
+def handle(inputs: Input) -> Optional[Output]:
```

Comment:
why output is optional? We have to have Output object

Reply:
inputs, theoretically, could be None in an error case

```
    if inputs.is_empty():
        # initialization request
        return None
```

Reply:
None is a special treatment we deal at the back, I don't feel like making Optional is a good idea,

Does

```
Union[None, Output] = Optional[Output]
```
?

Reply:
Yes, the two are the same thing. Optional is just syntactic sugar

Reply:
It's fine to remove the Optional - there won't be an error thrown since I'm most likely doing type checking in another way that doesn't take information directly from type hints.

Reply:
@ydm-amazon one good thing of Union is that it support 3.8

Reply:
Oh, what I meant by not supporting 3.8 is choosing to not do

`from typing import List` and using `List` as a type hint

and instead just using `list` as a type hint. This is a feature introduced in 3.9, but the former way is either already deprecated or soon to be deprecated